### PR TITLE
INSTUI-2873 New contribution process documentation

### PR DESCRIPTION
**Pull request by matyasf**
Also deleted the old RFCs, they are not relevant since a long while and I doubt there is any useful information in them for the current state of InstUI

### CODE REVIEW

File: docs/getting-started/contributing.md [hunk 1]
```diff
@@ -8,12 +8,25 @@ order: 2
...
+### What can you contribute?
 
-Follow these steps to download and run Instructure-UI on your machine:
+If you found a bug or want to add a minor feature, go ahead! If it's a larger thing (like a new component), read on.
+
+We try to keep the InstUI easy to use, easy to learn and free from bloat.
+For this reason we will only add new components if it fits into the current design patterns, and the new component cannot be
+constructed from the existing ones. For these reasons adding a new component is the following process:
+
+- You should write a document what do you want to add and what use case does it solve.
```
Comment (HerrTopi) [inline docs/getting-started/contributing.md:19]:
```suggestion
- You should write a document which includes the component description and the use cases it's meant to be used in.
```

File: docs/getting-started/contributing.md [hunk 2]
```diff
@@ -8,12 +8,25 @@ order: 2
...
+- You should make some kind of design sketch that documents the design of the pattern (e.g. with [Figma](https://www.figma.com/)).
```
Comment (HerrTopi) [inline docs/getting-started/contributing.md:20]:
```suggestion
- You should make some kind of design sketch that documents the design of the component (e.g. with [Figma](https://www.figma.com/)).
```

File: docs/getting-started/contributing.md [hunk 3]
```diff
@@ -8,12 +8,25 @@ order: 2
...
+- Our design team will review and approve this design sketch.
+- If the sketch is OK, you will need to make a pull request (read on for our code guidelines), the InstUI dev team and our a11y team will review the code and merge it.
+
+### Building InstUI from the source
+
+Follow these steps to download and compile Instructure-UI on your machine:
```
Comment (HerrTopi) [inline docs/getting-started/contributing.md:26]:
```suggestion
Follow these steps to download and compile Instructure UI on your machine:
```

**Review by HerrTopi — APPROVED** [review]
I left some grammar and styling suggestions in the review, but my english isn't perfect, so maybe these notes aren't meant to be taken at face value.

File: docs/getting-started/contributing.md [hunk 4]
```diff
@@ -8,12 +8,25 @@ order: 2
 
 Contributions are welcome! If you want to submit a change you will need to open a pull request on Github. To do so first fork the repository to your own Github account.
 
-**Installation steps:**
+### What can you contribute?
 
-Follow these steps to download and run Instructure-UI on your machine:
+If you found a bug or want to add a minor feature, go ahead! If it's a larger thing (like a new component), read on.
```
Comment (ke1k0) [inline docs/getting-started/contributing.md:13]:
```suggestion
If you found a bug or want to add a minor feature, go ahead!

If you'd like to add a new component, or change/add new visuals, please read on.
```

File: docs/getting-started/contributing.md [hunk 5]
```diff
@@ -8,12 +8,25 @@ order: 2
...
+
+We try to keep the InstUI easy to use, easy to learn and free from bloat.
+For this reason we will only add new components if it fits into the current design patterns, and the new component cannot be
+constructed from the existing ones. For these reasons adding a new component is the following process:
+
+- You should write a document what do you want to add and what use case does it solve.
+- You should make some kind of design sketch that documents the design of the pattern (e.g. with [Figma](https://www.figma.com/)).
+- Our design team will review and approve this design sketch.
+- If the sketch is OK, you will need to make a pull request (read on for our code guidelines), the InstUI dev team and our a11y team will review the code and merge it.
```
Comment (ke1k0) [inline docs/getting-started/contributing.md:22]:
```suggestion
- If the sketch is OK, you will need to make a pull request (read on for our code guidelines).
- The InstUI dev team and our a11y team will review the code and merge it.
```

**Pull request merged**